build.debug.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+BUILD_DIR=./build.dev
+
+FORCE_BUILD=false
+if [ "$1" = "-f" ]; then
+  FORCE_BUILD=true
+fi
+
+await_confirm() {
+  if ! $FORCE_BUILD; then
+    echo ""
+    echo "   To build using these settings, hit ENTER"
+    read confirm
+  fi
+}
+
+exit_message() {
+  echo "--------------------------------------------------------"
+  echo "Done. To install DASH, run    make install    in $BUILD_DIR"
+}
+
+if [ "${PAPI_HOME}" = "" ]; then
+  PAPI_HOME=$PAPI_BASE
+fi
+
+# To specify a build configuration for a specific system, use:
+#
+#                    -DENVIRONMENT_TYPE=<type> \
+#
+# For available types, see the files in folder ./config.
+# To specify a custom build configuration, use:
+#
+#                    -DENVIRONMENT_CONFIG_PATH=<path to cmake file> \
+#
+
+# To use an existing installation of gtest instead of downloading the sources
+# from the google test subversion repository, use:
+#
+#                    -DGTEST_LIBRARY_PATH=${HOME}/gtest \
+#                    -DGTEST_INCLUDE_PATH=${HOME}/gtest/include \
+#
+
+# To build with MKL support, set environment variables MKLROOT and INTELROOT.
+#
+
+# To enable IPM runtime support, use:
+#
+#                    -DIPM_PREFIX=<IPM install path> \
+
+# To build with MKL support, set environment variables MKLROOT and INTELROOT.
+
+# For likwid support, ensure that the likwid development headers are
+# installed.
+
+# Configure with default debug build settings:
+mkdir -p $BUILD_DIR
+rm -Rf $BUILD_DIR/*
+(cd $BUILD_DIR && cmake -DCMAKE_BUILD_TYPE=Debug \
+                        -DENVIRONMENT_TYPE=default \
+                        -DINSTALL_PREFIX=$HOME/opt/dash-0.3.0-dev/ \
+                        -DDART_IMPLEMENTATIONS=mpi \
+                        -DENABLE_THREADSUPPORT=OFF \
```

Comment:
No particular reason, I just copied it from the other build scripts,
but I can activate it for them as well.